package.json

```
@@ -74,6 +73,9 @@
     "stylelint": "^16.9.0",
     "table": "^6.8.1"
   },
+  "peerDependencies": {
+    "@primer/primitives": "^9.0.3"
```

```suggestion
    "@primer/primitives": ">=9.0.3"
```

Just thinking of ways we can make it not be too restrictive

---

I'm ok with `^` for now since I think we prob do want to make sure major bumps don't break primer/css.